lms/djangoapps/edraak_university/views.py

```diff
@@ -57,6 +58,8 @@ def get(self, *args, **kwargs):
         if self.is_staff():
             return redirect('edraak_university:id_staff', course_id=course.id)
 
+        CourseTabView.register_user_access_warning_messages(self.request, self.get_course_key())
```

Comment:
Also this one. Not super complicated, but something basic to ensure it's not lost during future merges.

Reply:
Done :+1: